dlp/dlp_inspect_image_file.rb

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# [START dlp_inspect_image_file]
+require "google/cloud/dlp"
+
+def inspect_image_file (
```

Comment:
Comparing this with the [Node.js PR](https://github.com/googleapis/nodejs-dlp/pull/154)

I really like the use of **_standard, idiomatic_** documentation of the method and its parameters using _conventional_ method documentation (RDoc), which the Node.js ([snippet](https://github.com/googleapis/nodejs-dlp/blob/787b237bb214dfadac83ccec96aec25010f27d1f/samples/dlp_inspect_image_file.js#L25-L32)) and C# ([snippet](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/aa405e94f5e19da8d108336bffe5ddb2ac0ee0b6/dlp/api/Snippets/DeidentifyMasking.cs#L23-L30)) proposed templates both use.

**Recommend:** we do the same for Ruby. It's a better (idiomatic) way of documenting our variables which all Rubyists are accustomed to 👍

```ruby
# Inspects the provided text for sensitive data.
#
# @param [String] project_id Your Google Cloud Project ID.
# @param [String] filepath The path to the image file to inspect.
def inspect_image_file(...)
```

```ruby
# Inspects the provided text for sensitive data.
#
# @param [String] project_id Your Google Cloud Project ID.
# @param [String] text_to_inspect The text to inspect.
def inspect_string(...)
```

Reply:
See https://yardoc.org for documentation on the yardoc format. Also, see the https://github.com/googleapis/google-cloud-ruby repo for lots of examples. (This is our canonical repo for client libraries.)